### DISCUSSION TITLE
Auto refresh with python script

### DISCUSSION THREAD

**Discussion by d-ariho** [Azure/CCOInsights#272] [Q&A] [answered]
Would it be possible to use an auto refresh script such as the pbixrefresher python script found here? https://github.com/dubravcik/pbixrefresher-python
Just as a bypass until the package is modified. Me and my team are trying to automate the dashboard refresh as it does take time doing it manually for our tenant.
If it is possible, would anyone have any idea about how we could go about automating entering credentials for separate tenants. Not having to enter them manually which may affect the script.

**Comment by josunefon** [comment, marked as answer]
It's possible but we never tested this solution with the CCO Insights Dashboard, you can create a [feature request ](https://github.com/Azure/CCOInsights/issues/new/choose) and the team can take a look at it.
